List/LocationList.py

```diff
@@ -275,45 +275,45 @@ def getTreasureList():
+            KH2Treasure(582, "CoR Mineshaft Upper Level Magic Boost", LocationTypes=[locationType.HB], InvalidChecks=[] if trueRandom else [itemType.FORM, itemType.GROWTH_ABILITY]).setLocationWeight(10),
             #last chest
-            KH2Treasure(579, "CoR Mineshaft Upper Level AP Boost", LocationTypes=[locationType.HB, locationType.CoR], InvalidChecks=[itemType.FORM, itemType.GROWTH_ABILITY]),
-            KH2Treasure(560, "Demyx (Data) AP Boost", LocationTypes=[locationType.HB, locationType.DataOrg], InvalidChecks=[itemType.FORM, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(588, "Winner's Proof", LocationTypes=[locationType.HB], InvalidChecks=[itemType.FORM, itemType.THUNDER, itemType.MAGNET, itemType.TORN_PAGE, itemType.PROOF_OF_PEACE, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(589, "Proof of Peace", LocationTypes=[locationType.HB], InvalidChecks=[itemType.FORM, itemType.THUNDER, itemType.MAGNET, itemType.TORN_PAGE, itemType.PROOF_OF_PEACE, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
+            KH2Treasure(579, "CoR Mineshaft Upper Level AP Boost", LocationTypes=[locationType.HB], InvalidChecks=[] if trueRandom else [itemType.FORM, itemType.GROWTH_ABILITY]).setLocationWeight(10),
+            KH2Treasure(560, "Demyx (Data) AP Boost", LocationTypes=[locationType.HB, locationType.DataOrg], InvalidChecks=[itemType.FORM, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(588, "Winner's Proof", LocationTypes=[locationType.HB], InvalidChecks=([] if trueRandom else [itemType.FORM, itemType.THUNDER, itemType.MAGNET, itemType.TORN_PAGE])+[itemType.PROOF_OF_PEACE, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(589, "Proof of Peace", LocationTypes=[locationType.HB], InvalidChecks=([] if trueRandom else [itemType.FORM, itemType.THUNDER, itemType.MAGNET, itemType.TORN_PAGE])+[itemType.PROOF_OF_PEACE, itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
         
-            KH2Treasure(492, "Gorge Savannah Map", LocationTypes=[locationType.PL]),
-            KH2Treasure(404, "Gorge Dark Gem", LocationTypes=[locationType.PL]),
-            KH2Treasure(405, "Gorge Mythril Stone", LocationTypes=[locationType.PL]),
+            KH2Treasure(492, "Gorge Savannah Map", LocationTypes=[locationType.PL]).setLocationWeight(-10),
+            KH2Treasure(404, "Gorge Dark Gem", LocationTypes=[locationType.PL]).setLocationWeight(-10),
+            KH2Treasure(405, "Gorge Mythril Stone", LocationTypes=[locationType.PL]).setLocationWeight(-10),
             KH2Treasure(401, "Elephant Graveyard Frost Gem", LocationTypes=[locationType.PL]),
```

Comment:
I think all of the chests in PL should have early game weight

Reply:
I have seen some players have trouble with the first fight, so I was being cognizant of that. Willing to change though if you disagree.

Reply:
the mission statement of "everything before any fights is easy" makes sense, we can leave it like that until the community gets some time with it